drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -290,8 +290,12 @@ function DragAndDropTemplates(configuration) {
             h('div.keyboard-help-dialog', [
                 h('div.modal-window-overlay'),
                 h('div.modal-window', {attributes: {role: 'dialog', 'aria-labelledby': labelledby_id}}, [
+                    h('button.modal-dismiss-button.unbutton', {attributes: {tabindex: 1}}, [
```

Comment:
This button should not have a tabindex set (buttons are naturally focusable). You can check for errors like this using Chrome's accessibility audit tools, or the Axe Chrome plugin.

![image](https://cloud.githubusercontent.com/assets/484484/21116963/ce11a8d0-c085-11e6-84c4-fde53eec8734.png)